### DISCUSSION TITLE
Sticky pistons feature

### DISCUSSION THREAD

**Discussion by MrRaposinha** [PurpurMC/Purpur#715] [__Dismissed__] [answered]
Just as it's possible to use a honeycomb on a copper block, it would be nice if we could use a slime ball on a regular piston to change them to a sticky piston.

(Sorry if something is wrong, my english is not the best.)

**Comment by encode42** [comment, marked as answer]
Easier to implement as a plugin, and honestly a nice starter plugin!

On player interact → if interacted block is piston → if player holding slime → replace piston with sticky piston
